script/test/cmd/cmd_test.go

```diff
@@ -0,0 +1,248 @@
+		return nil
+	}
+	return os.ErrPermission
+}
+
+func binary(cmd string) string {
+	path, err := exec.LookPath(cmd)
+	if err != nil || fileNotExist(cmd) {
+		home := os.Getenv("HOME")
+		if home == "" {
+			fmt.Println("No $HOME environment variable found")
+		}
+	}
+	return path
+}
+
+// Kompose command
+func Kompose(command, provider, dockerComposeFile, result string) (cmd, output string) {
+	b, _ := ioutil.ReadFile(result)
+	file := strings.Split(dockerComposeFile[9:], "/")
+	output = string(b)
+	cmdName := "kompose"
+	cmdArgs := []string{command, "--provider", provider, "-f", dockerComposeFile, "--stdout", "-j"}
+	out, err := exec.Command(cmdName, cmdArgs...).CombinedOutput()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Error Cmd", err, file[0])
+		os.Exit(1)
+	}
+	cmd = string(out)
+	return cmd, output
+}
+
+// Compare compares two json files
+func Compare(output, cmd, dockerComposeFile, provider string) error {
+	file := strings.Split(dockerComposeFile[9:], "/")
+	if reflect.DeepEqual(output, cmd) {
+		fmt.Println("===> Starting Test ")
+		fmt.Println("Testing", file[0], "for", provider)
+		logrus.Info("\x1b[32;1mTest Pass\x1b[0m")
+	} else {
+		fmt.Println("\x1b[31;1m===> Test Fail <===\x1b[0m")
+		logrus.Error(provider, " \x1b[31;1mTest Fail\x1b[0m for ", file[0])
+	}
+	fmt.Println("")
+	return nil
+}
+
+// ExpectSuccess Test for K8s
+func ExpectSuccess(command, provider, dockerComposeFile, result string) {
+	cmd, output := Kompose(command, provider, dockerComposeFile, result)
+	Compare(output, cmd, dockerComposeFile, provider)
+}
+
+// ExpectSuccessAndWarning Test
+func ExpectSuccessAndWarning(command, provider, dockerComposeFile, result string) {
+	warning := "WARN"
+	info := "INFO"
+	cmd, output := Kompose(command, provider, dockerComposeFile, result)
+	if strings.Contains(cmd, warning) || strings.Contains(cmd, info) {
+		cmdName := "kompose"
+		cmdArgs := []string{command, "--provider", provider, "--stdout", "-f", dockerComposeFile, "-j"}
+		str, err := exec.Command(cmdName, cmdArgs...).Output()
+		if err != nil {
+			fmt.Fprintln(os.Stderr, "Err Cmd", err)
+			os.Exit(1)
+		}
+		cmd := string(str)
+		Compare(output, cmd, dockerComposeFile, provider)
+	}
+}
+
+// ExpectFailure Test
+func ExpectFailure(command, provider, dockerComposeFile string) {
+	cmdName := "kompose"
+	cmdArgs := []string{command, provider, "--stdout", "-f", dockerComposeFile, "-j"}
+	_, err := exec.Command(cmdName, cmdArgs...).CombinedOutput()
+	file := strings.Split(dockerComposeFile[9:], "/")
+	if err != nil {
+		fmt.Println("===> Starting Test ")
+		fmt.Println("Testing", file[0], "for", provider)
+		logrus.Info("\x1b[32;1mTest Pass\x1b[0m")
+	} else {
+		fmt.Println("\x1b[31;1m===> Test Fail <===\x1b[0m")
+		logrus.Error(provider, " \x1b[31;1mTest Fail\x1b[0m for", file[0])
+	}
+	fmt.Println("")
+}
+
+// LoadEnv files
+func LoadEnv(envFile string) {
+	err := godotenv.Overload(envFile)
+	if err != nil {
+		logrus.Error("Unable to load env file")
```

Comment:
@containscafeine I tried using `error.Wrap()` here, don't know if I used it in the right way or not.
```go
func LoadEnv(envFile string) error {
	err := godotenv.Overload(envFile)
	if err != nil {
		error.Wrap(err, "Unable to load env file")
	}
        return nil
}
```